### ISSUE TITLE
Implicit cast warning

### ISSUE THREAD

**Issue by mahdi13** [Legrandin/pycryptodome#55]
When I : 
```
public_key = new_key.publickey().exportKey('PEM')
```

I get :

```
lib/python3.5/site-packages/Cryptodome/Math/_Numbers_gmp.py:230: UserWarning: implicit cast to 'char *' from a different pointer type: will be forbidden in the future (check that the types are as you expect; use an explicit ffi.cast() if they are correct)
  _gmp.gmp_snprintf(buf, c_size_t(buf_len), b("%Zd"), self._mpz_p)
```

version : `pycryptodomex==3.4.3`
@pylover

**Comment by Legrandin** [comment]
Thanks for reporting!
It has actually been fixed recently (0015a57e41b328c85d24854a65d4fe2fbc768f79) and it ended up in the latest release.

**Comment by pylover** [comment]
Thanks

**Comment by mahdi13** [comment]
@Legrandin hasn't you released 3.4.5 yet? I couldn't find it.
it is lastest version:
[](https://pypi.python.org/pypi/pycryptodomex/3.4.3)

**Comment by Legrandin** [comment]
@mahdi13 The PyCryptodomex variant had been falling behind a bit. My bad. Now, it is up to date:

https://pypi.python.org/pypi/pycryptodomex/3.4.5
